src/Ray.hs

```diff
@@ -18,3 +18,21 @@ rayMarch s end (pos,dir) -- End is the render distance (how far to march before
     | dist < 0.0001 = Just $ Vec3 1 0 0 -- Test Color-TODO
     | otherwise = rayMarch s (end-dist) (pos + dist `scale` dir, dir) -- Each time lowering the distance to the end with the distance we traveled.
     where   (dist, _) = s pos
+
+
+getXComponents :: Int -> [Double]
+getXComponents width = take width [-1, -1 + 2 / fromIntegral width .. ] -- Generates equally spaced floats from -1 to 1.
+
+widthCoords :: [Double]
+widthCoords = getXComponents 1024 -- Width of the image
```

Comment:
Constants are not acceptable.

Reply:
If we later want this to not use a constant we'd have to change the function to `Settings -> [Double]`, so it might as well start off as that.